modules/tinymce/src/plugins/table/test/ts/browser/FakeSelectionTest.ts

```diff
@@ -110,4 +110,67 @@ describe('browser.tinymce.plugins.table.FakeSelectionTest', () => {
     editor.setContent('<table><tr><td data-mce-selected="1">a</td><td>b</td></tr><tr><td data-mce-selected="1">c</td><td>d</td></tr></table>', { format: 'raw' });
     assertSelectionContent(editor, '<table><tbody><tr><td>a</td></tr><tr><td>c</td></tr></tbody></table>');
   });
+
+  it('TINY-7724: can select single CEF cell', () => {
+    const editor = hook.editor();
+    assertTableSelection(
+      editor,
+      '<table><tr><td contenteditable="false">1</td><td>2</td></tr><tr><td>3</td><td>4</td></tr></table>',
+      [ '1', '1' ],
+      [ '1' ]
+    );
+    TinyAssertions.assertContentPresence(editor, {
+      'td[contenteditable="false"][data-mce-selected="1"][data-mce-first-selected="1"][data-mce-last-selected="1"]': 1
+    });
+    TinyAssertions.assertSelection(editor, [ 0, 0, 0 ], 0, [ 0, 0, 0 ], 1);
+  });
+
+  it('TINY-7724: can select CEF cell and normal cell', () => {
+    const editor = hook.editor();
+    assertTableSelection(
+      editor,
+      '<table><tr><td contenteditable="false">1</td><td>2</td></tr><tr><td>3</td><td>4</td></tr></table>',
+      [ '1', '3' ],
+      [ '1', '3' ]
+    );
+  });
+
+  it('TINY-7724: does not select CEF cell if contenteditable=true child is selected', () => {
+    const editor = hook.editor();
+    editor.setContent('<table><tr><td contenteditable="false"><p contenteditable="true">1</p></td><td>2</td></tr><tr><td>3</td><td>4</td></tr></table>');
+    TinySelections.setCursor(editor, [ 0, 0, 0, 0, 0, 0 ], 0);
```

Comment:
Is this needed if we then click on the paragraph before doing any assertions in this state?

Reply:
I thought it was needed but looks like it isn't necessary so have removed it. Thanks.